src/components/Quiz/QuizzesStats.tsx

```diff
@@ -172,7 +214,7 @@ const QuizzesStats: React.FC = () => {
               </Text>
 
               {/* Data from Matomo, manually updated */}
-              <Text color="body">100 000+</Text>
+              <Text color="body">{textForCollectiveQuestions}</Text>
```

Comment:
If you use this, you can skip the "+" logic above.
```suggestion
              <Text color="body" _after={{ content: `"+"` }}>
                {formattedCollectiveQuestionsAnswered}
              </Text>
```

Reply:
Ahh nice thinking, I hadn't thought of this. Agree that the code is pretty convoluted and open to ideas on how to make it less so 😁

One potential problem with using `after` is that pseudo-element elements are ignored by screen readers.

Reply:
Good point. I'd say that if we care about sr then we could use a inner span element. This will get placed at the beginning if `ltr`.
```suggestion
              <Text color="body">
                {formattedCollectiveQuestionsAnswered}
                <Text as="span">+</Text>
              </Text>
```